Packs/TOPdesk/Integrations/TOPdesk/TOPdesk_test.py

```diff
@@ -943,7 +948,7 @@ def test_fetch_incidents(client, requests_mock, topdesk_incidents_override, last
         if incident_override['will_be_fetched']:
             expected_incidents.append({
                 'name': f"{incident_override['briefDescription']}",
-                'details': json.dumps(response_incident),
+                'details': f"{response_incident['request']}",
```

Comment:
Why was this changed?

Reply:
While I was comparing the code with JiraV2.py in fetch_incidents and create_incidents, I saw the incident there also puts the 'demisto.get(issue, "fields.description")' in the details field. It makes more sense, because before the complete json data was put there, while it was also stored in the rawJSON fields. It sounds like the same data on two places and no good description for the incident.

I see ServiceNow is dumping the complete JSON there also. So it is not consistent within all the fetch_incidents.

But the description sounds more logical and shows better in the layout.